client/data/domains/nameservers/use-domain-nameservers-query.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import { useQuery } from 'react-query';
+
+/**
+ * Internal dependencies
+ */
+import wp from 'calypso/lib/wp';
+
+const useDomainNameserversQuery = ( domainName, queryOptions = {} ) =>
+	useQuery(
+		[ 'domain-nameservers', domainName ],
+		() => wp.req.get( `/domains/${ domainName }/nameservers` ),
```

Comment:
One nit: in the [previous version of the code](https://github.com/Automattic/wp-calypso/pull/53657/files#diff-df294adbc1c8e1ee954d8808da3aeeb2944bcbfb702eb467da521e3e2a4e63e8L24), we were making requests to those endpoints with a trailing slash. Here, we don't have a trailing slash. Seems to me that it doesn't make any difference, but just wanted to bring it up.

Reply:
fixed with https://github.com/Automattic/wp-calypso/pull/53657/commits/a373cf1cb96bda8117ca9b5838ffdbe4b0182152